components/PortfolioContent.json

```
@@ -5,27 +5,45 @@
         "name":"GoodreadR",
         "desc1": "GoodReadR is an app for online bookstore owners. It adds a Goodreads review widget on your Shopify store",
         "image1":"../assets/img/work/p1.png",
-        "site": "https://apps.shopify.com/goodreadr-book-reviews-1"
+        "site": "https://apps.shopify.com/goodreadr-book-reviews-1",
+        "tech1": "Koa (Node.js)",
+        "tech2": "React",
+        "tech3": "Shopify Polaris",
+        "tech4": "Upstash Redis"
```

It would be more elegent if we make this into an array.

```suggestion
        "tech": ["Koa (Node.js)", "React", Shopify Polaris", "Upstash Redis"]
```